launcher/ui/pages/modplatform/ModPage.cpp

```diff
@@ -129,13 +131,19 @@ void ModPage::updateVersionList()
                 break;
             }
         }
-
+        QString flag;
+        if (installedIndex == -1 && installedVersion.isValid() && installedVersion == version.fileId) {
+            flag = tr("[installed]");
+            installedIndex = i;
+        }
         // Only add the version if it's valid or using the 'Any' filter, but never if the version is opted out
         if ((valid || m_filter->versions.empty()) && !optedOut(version)) {
             auto release_type = version.version_type.isValid() ? QString(" [%1]").arg(version.version_type.toString()) : "";
-            m_ui->versionSelectionBox->addItem(QString("%1%2").arg(version.version, release_type), QVariant(i));
+            m_ui->versionSelectionBox->addItem(QString("%1%2%3").arg(version.version, release_type, flag), QVariant(i));
```

Comment:
```suggestion
            m_ui->versionSelectionBox->addItem(QString("%1%2 %3").arg(version.version, release_type, flag), QVariant(i));
```
the space could also go in the original variable, but things like `[release][installed]` is kinda ugly to me

Reply:
I understand that the current flags are ugly, but I have negative UX skills so if you make any kind of suggestions please be very specific. For now added the space before it and also moved and renamed the button.